www/install/php/Update-22.04.0-beta.1.php

```diff
@@ -458,3 +461,135 @@ function updateSecurityPolicyConfiguration(CentreonDB $pearDB): void
     $statement->bindValue(':localProviderConfiguration', $localProviderConfiguration, \PDO::PARAM_STR);
     $statement->execute();
 }
+
+/**
+ * Migrate broker outputs 'sql' and 'storage' to a unique output 'unified_sql'
+ *
+ * @param CentreonDB $pearDB
+ * @return void
```

Comment:
Don't forget to declare exceptions
```suggestion
 * @throws \Exception
 * @return void
```